src/utils/vedleggUtils.ts

```diff
@@ -47,10 +47,6 @@ function opprettFormDataMedVedlegg(metadata: Metadata[]): FormData {
     return formData;
 }
 
-export function containsUloveligeTegn(filnavn: string, uloveligeTegnList: string[]) {
-    for (const uloveligTegn of uloveligeTegnList) {
-        if (filnavn.indexOf(uloveligTegn) !== -1)
-            return true;
-    }
-    return false;
+export function containsUlovligeTegn(filnavn: string) {
+    return filnavn.match(new RegExp("[^a-zæøåA-ZÆØÅ0-9 ._-]"));
```

Comment:
Vi lurte litt på om det er andre tegn som også bør støttes, eks ```,```, men det er jo ikke veldig viktig.

Reply:
Jeg spurte Joachim og han godkjente. Wilhelm sa heller ingenting på det. så jeg tror vi kjører på. Hadde kanskje vært kjekt å ha en logg for å se hvilke tegn som ikke blir støttet.

Reply:
Ja, dette er jo de mest brukte tegnene. :)

Reply:
La til , ( og )

Reply:
Pluss en bindestrek til. finnes jo en kort og en lang en